packages/gatsby-telemetry/src/index.ts

```diff
@@ -24,35 +29,84 @@ function tick(): void {
     .then(() => setTimeout(tick, interval))
 }
 
-module.exports = {
-  trackFeatureIsUsed: (name: string): void => instance.trackFeatureIsUsed(name),
-  trackCli: (input, tags, opts): void =>
-    instance.captureEvent(input, tags, opts),
-  trackError: (input, tags): void => instance.captureError(input, tags),
-  trackBuildError: (input, tags): void =>
-    instance.captureBuildError(input, tags),
-  setDefaultTags: (tags): void => instance.decorateAll(tags),
-  decorateEvent: (event, tags): void => instance.decorateNextEvent(event, tags),
-  setTelemetryEnabled: (enabled): void => instance.setTelemetryEnabled(enabled),
-  startBackgroundUpdate: (): void => {
-    setTimeout(tick, interval)
-  },
-  isTrackingEnabled: (): boolean => instance.isTrackingEnabled(),
-  aggregateStats: (data): IAggregateStats => instance.aggregateStats(data),
-  addSiteMeasurement: (event, obj): void =>
-    instance.addSiteMeasurement(event, obj),
-  expressMiddleware: function (source: string) {
-    return function (
-      _req: express.Request,
-      _res: express.Response,
-      next
-    ): void {
-      try {
-        instance.trackActivity(`${source}_ACTIVE`)
-      } catch (e) {
-        // ignore
-      }
-      next()
+export function trackFeatureIsUsed(name: string): void {
+  instance.trackFeatureIsUsed(name)
+}
+export function trackCli(
+  input: string | string[],
+  tags?: ITelemetryTagsPayload,
+  opts?: ITelemetryOptsPayload
+): void {
+  instance.captureEvent(input, tags, opts)
+}
+
+export function trackError(input, tags): void {
+  instance.captureError(input, tags)
+}
+
+export function trackBuildError(input, tags): void {
+  instance.captureBuildError(input, tags)
+}
+
+export function setDefaultTags(tags): void {
+  instance.decorateAll(tags)
+}
+
+export function decorateEvent(event, tags): void {
+  instance.decorateNextEvent(event, tags)
+}
+
+export function setTelemetryEnabled(enabled): void {
+  instance.setTelemetryEnabled(enabled)
+}
+
+export function startBackgroundUpdate(): void {
+  setTimeout(tick, interval)
+}
+
+export function isTrackingEnabled(): boolean {
+  return instance.isTrackingEnabled()
+}
+
+export function aggregateStats(data): IAggregateStats {
+  return instance.aggregateStats(data)
+}
+
+export function addSiteMeasurement(event, obj): void {
+  instance.addSiteMeasurement(event, obj)
+}
+
+export function expressMiddleware(source: string) {
+  return function (_req: express.Request, _res: express.Response, next): void {
+    try {
+      instance.trackActivity(`${source}_ACTIVE`)
+    } catch (e) {
+      // ignore
     }
-  },
+    next()
+  }
+}
+
+// Internal
+export function setDefaultComponentId(componentId): void {
+  instance.componentId = componentId
+}
+
+export function setGatsbyCliVersion(version): void {
+  instance.gatsbyCliVersion = version
+}
```

Comment:
Looks like these weren't exported before. These names do not show up in this PR. What's the reason for their export in this PR?

Reply:
whacky way to allow the same parameters to be updated as with the constructor. I don't particularly like this but considering the index.ts implementation these kind of functions seemed like the easiest solution

Reply:
ofc these could me made to use setters instead of poking the object directly

Reply:
The question was more around exposing a function that isn't used. Sort of like a premature optimization / overengineering. Could be valid, just wanted to double check. Feel free to merge otherwise. It's all greens :)